alf/networks/encoding_networks.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) 2020 Horizon Robotics. All Rights Reserved.
```

Comment:
This is in the other PR #366. You either make 1 PR or have separate changes in 2 PRs.

Reply:
Yeah this PR depends on #366. I can remove the files, but it will make the review more difficult (you have to look at code in another PR).